userspace/falco/webserver.h

```diff
@@ -16,13 +16,14 @@ limitations under the License.
 */
 #pragma once
 
-#define CPPHTTPLIB_OPENSSL_SUPPORT
-#define CPPHTTPLIB_ZLIB_SUPPORT
```

Comment:
These two defines were resulting in a warning that the definition was already present.